oauth2client/contrib/appengine.py

```diff
@@ -210,6 +211,35 @@ def create_scoped_required(self):
     def create_scoped(self, scopes):
         return AppAssertionCredentials(scopes, **self._kwargs)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
```

Comment:
Perhaps I shouldn't have mentioned Java... but since I did, here's what happens there: if you leave an overriding method undocumented, Javadoc renders the overridden documentation in its place. So the documentation only has to exist one place in code and its visible in two (or N, as the case might be).

We don't have that for Python, so what I'm pushing for is the "only maintain the documentation in one place" option, coming as it does at the cost of "users have to navigate from one place to another". I think the dangers of having to maintain multiple copies of text generally outweigh the inconvenience of users having to manually navigate from one method to another, so I've adopted a personal rule and habit around it, and am applying that rule here. I see that you feel otherwise in this particular case, but do you also feel otherwise in general?